src/Generator/Parsing/DocumentationParser.cs

```diff
@@ -37,23 +47,36 @@ public static Documentation Parse(DocumentationSource source)
                     PropertyInfo? propertyInfo = reader.GetType().GetProperty("DisableUndeclaredEntityCheck", BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic);
                     propertyInfo!.SetValue(reader, true);
                     XDocument xml = XDocument.Load(reader);
-                    CommandDocumentation[] documentation = ParseFile(xml.Root!);
+
+                    string refPagesLink = "https://www.khronos.org/registry/OpenGL-Refpages/";
+                    string filename = Path.GetFileNameWithoutExtension(file.Name);
+                    switch (api)
+                    {
+                        case OutputApi.GL:
+                            refPagesLink += $"gl4/html/{filename}.xhtml";
+                            break;
+                        case OutputApi.GLCompat:
+                            refPagesLink += $"gl2.1/xhtml/{filename}.xml";
+                            break;
+                        case OutputApi.GLES1:
+                            refPagesLink += $"es1.1/xhtml/{filename}.xml";
+                            break;
+                        case OutputApi.GLES3:
+                            refPagesLink += $"es3.0/html{filename}.xhtml";
+                            break;
+                        default:
+                            refPagesLink = "NOT FOUND";
```

Comment:
I would avoid adding spaces into a URL, would be better to add something like `not-found` or something that is more url friendly.